packages/client/src/comp/states/Base.tsx

```diff
@@ -52,6 +52,7 @@ const ContainerlessBase = <Info extends StateInfo>({ enrichValues, children }: P
     canCheckRegistration,
     validInputs,
   } = FieldsContainer.useContainer()
+  const [ alloyId, setAlloyId ] = React.useState<string>()
```

Comment:
Would it be easier to have `React.useState< AlloyResponse>`?

Reply:
Certainly, I'll make these changes